polynote-frontend/polynote/interpreter/client_interpreter.ts

```diff
@@ -145,7 +145,7 @@ export function availableResultValues(symbols: KernelSymbols, cellOrder: number[
 
     return whichCells.reduce<Record<string, ResultValue>>((acc, next) => {
         Object.values(symbols[next] || {})
-            .forEach((result: ResultValue) => acc[result.name] = result);
+            .forEach((result: ResultValue) => acc[sanitizeJSVariable(result.name)] = result);
```

Comment:
talked with @jeremyrsmith and it turns out this function is used by the symbol table too. So: 
1. Should move this function somewhere else (maybe notebook_state)? 
2. Move the sanitization over to Vega Interpreter directly.